src/lib/commit-message.ts

```diff
@@ -101,10 +110,14 @@ export function serializeHeader(partialCommitMessage: {
 
 export function serialize(commitMessage: CommitMessage) {
   let message = serializeHeader(commitMessage);
-  const { body, footer } = commitMessage;
+  const { body, breakingChange, footer } = commitMessage;
   if (body) {
     message += `\n\n${body}`;
   }
+  if (breakingChange) {
+    message = message.replace(': ', '!: ');
```

Comment:
We can have the `!:` implemented in the `serializeHeader` function, other than `serializeHeader` first and `replace` after.